### PR TITLE
Implement deleting a post

### PR DESCRIPTION
**Pull request by kazeens**

### CODE REVIEW

File: src/features/Posts/actions.js [hunk 1]
```diff
@@ -27,3 +27,7 @@ export const addPost = postData => dispatch => {
     })
 };
 
+export const deletePost = postId => dispatch => rest.api.deletePost(postId)
+  .then(() => {
+    dispatch(fetchPosts());
+  });
```
Comment (Starotitorov) [inline src/features/Posts/actions.js]: Add new line at the end of file

File: src/features/Posts/PostsList/Post.js [hunk 2]
```diff
@@ -1,17 +1,33 @@
...
+  display: block;
+  color: black;
+  position: absolute;
+  top: 10px;
+  right: 10px;
+`;
+
+const CustomCard = styled(Card)`
+  position: relative;
+`;
```
Comment (Starotitorov) [inline src/features/Posts/PostsList/Post.js]: Add new line at the end of file.

File: src/features/Posts/PostsList/PostsList.js [hunk 3]
```diff
@@ -16,7 +16,7 @@ const PostsList = ({ posts, isLoading }) =>
         <div>
           {
             items.map(({ title, body, id }) =>
-              <CustomPost key={id} title={title} body={body}/>
+              <CustomPost deletePost={deletePost} key={id} id={id} title={title} body={body}/>
```
Comment (Starotitorov) [inline src/features/Posts/PostsList/PostsList.js]: Rename `deletePost` to `onDelete` callback.

File: src/features/Posts/PostsList/PostsList.js [hunk 4]
```diff
@@ -3,9 +3,9 @@ import PropTypes from 'prop-types';
 import { Alert } from 'reactstrap';
 import styled from 'styled-components';
 import { List, LoadingIndicator } from 'components';
-import Post from './Post';
+import Post from './PostView';
 
-const PostsList = ({ posts, isLoading }) =>
+const PostsList = ({ posts, isLoading, deletePost }) =>
   <List
```
Comment (Starotitorov) [inline src/features/Posts/PostsList/PostsList.js]: Please add deletePost to List properties and pass all props `children(props)` in List component

File: src/index.js [hunk 5]
```diff
@@ -5,6 +5,7 @@ import { injectGlobal } from 'styled-components';
 import App from './features/App';
 import registerServiceWorker from './registerServiceWorker';
 import { createStore } from './store';
+import 'font-awesome/css/font-awesome.min.css';
```
Comment (Starotitorov) [inline src/index.js]: Move this to App.js

File: src/features/Posts/actions.js [hunk 6]
```diff
@@ -27,3 +27,7 @@ export const addPost = postData => dispatch => {
     })
 };
 
+export const deletePost = postId => dispatch => rest.api.deletePost(postId)
+  .then(() => {
```
Comment (Starotitorov) [inline src/features/Posts/actions.js]: No need to write brackets. just
```
export const deletePost = postId => dispatch =>
  rest.api.deletePost(postId)
    .then(() => dispatch(fetchPosts()));
```

File: src/features/Posts/PostsList/PostsList.js (hunk 4 above)
Comment (kazeens) [reply src/features/Posts/PostsList/PostsList.js]: What do you exactly mean? Could you specify with example on ```children(props)```

Comment (Starotitorov) [reply src/features/Posts/PostsList/PostsList.js]: Look at List.js file

**Pull request closed without merging**